test/plugin/test_in_tail.rb

```diff
@@ -2886,22 +2911,20 @@ def test_updateTW_after_refreshTW
 
       assert_equal(
         {
-          # TODO: This is BUG!! We need to fix it and replace this with the next.
-          record_values: ["file1 log1", "file1 log1", "file1 log2", "file1 log2", "file2 log1", "file2 log2"],
-          # record_values: ["file1 log1", "file1 log2", "file2 log1", "file2 log2"],
-          tail_watcher_paths: ["#{@tmp_dir}/tail.txt", "#{@tmp_dir}/tail.txt", "#{@tmp_dir}/tail.txt1"],
-          tail_watcher_inodes: [inode_0, inode_1, inode_0],
-          tail_watcher_io_handler_opened_statuses: [false, false, false],
-          # TODO: This is BUG!! We need to fix it and replace this with the next.
+          record_values: ["file1 log1", "file1 log2", "file2 log1", "file2 log2"],
+
+          # TODO: Check if this is expected. 
+          tail_watcher_paths: ["#{@tmp_dir}/tail.txt", "#{@tmp_dir}/tail.txt"],
+          tail_watcher_inodes: [inode_0, inode_1],
+          tail_watcher_io_handler_opened_statuses: [false, false],
+          # tail_watcher_inodes: [inode_0, inode_1, inode_0],
+          # tail_watcher_paths: ["#{@tmp_dir}/tail.txt", "#{@tmp_dir}/tail.txt", "#{@tmp_dir}/tail.txt1"],
+          # tail_watcher_io_handler_opened_statuses: [false, false, false],
```

Comment:
I think we need to figure out if this is an expected behavior.

Reply:
Done as https://github.com/fluent/fluentd/compare/155bdcabfc61180c0880a47725952e1db5fea49a..bc130634eebc03c45d8ab1170f8e5a162110d311